zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/RecordIndexRouter.java

```diff
@@ -61,6 +62,10 @@ String aliasNameForValueType(final ValueType valueType) {
   /** Returns the index for this value type, minus the current date. */
   String indexPrefixForValueType(final ValueType valueType) {
```

Comment:
I think we shouldn't use this method anymore, as we always need to make sure that we write into the right version or?

Reply:
I looked at the other usages and I agree 
- tests 
- `RecordIndexRouter.searchPatternForValueType[81]` we want this to return the versionUtil version
- `ElasticsearchClient.putIndexTemplate[165]` we want this to return the versionUtil version as we are created the current exporter (8.7.0) indices. 

All this can be refactored such that the (ValueType, Version) is the only function that is used but just wanted to make sure it's the correct logic as a draft.